model/metrics/container.yaml

```diff
@@ -0,0 +1,52 @@
+groups:
+  - id: metric.container.cpu.utilization
+    type: metric
+    metric_name: container.cpu.utilization
+    brief: "Recent CPU utilization for the container."
+    note: >
+      CPU usage percentage normalized by the number of CPU cores.
+      The value range is [0.0,1.0].
+    instrument: gauge
+    unit: "1"
+    attributes:
+      - ref: container.id
```

Comment:
I see you've added only `container.id` as an attribute. Is there reasoning behind only including this? 

For reference, the docker stats receiver currently has the following resource attributes: 
```
rb.SetContainerRuntime("docker")
rb.SetContainerHostname(container.Config.Hostname)
rb.SetContainerID(container.ID)
rb.SetContainerImageName(container.Config.Image)
rb.SetContainerName(strings.TrimPrefix(container.Name, "/"))
rb.SetContainerImageID(container.Image)
rb.SetContainerCommandLine(strings.Join(container.Config.Cmd, " "))
```

Reply:
I'm not sure if we want to overload the metrics with all these in this first PR. Maybe we can add them as optional later if we see the need, since just `container.id` can be enough for signal correlation.
However I'm not sure if the implementations should strictly follow what is defined in the SemConv 🤔 .